css/base/utilities.css

```diff
@@ -33,6 +33,15 @@
 }
 
 @media screen and (max-width: 700px) {
+
+  .flex-col-sm {
+    flex-direction: column;
+  }
+
+  .flex-items-start-sm {
```

Comment:
Just a question why is there `-sm` in the class name.  Maybe you can use these classes for the large screens as well.

Reply:
These are just the responsive variants. For responsive screens, I required flex-items-center for larger screens and flex-items-start for smaller screens. For that reason, I have made utility classes in the respective media queries. That's why I have used `class="flex-items-center flex-items-start-sm"`. 
This is inspired from Bootstrap. You may check out the responsive variants for flex grid in the Bootstrap docs.

Reply:
Ok Cool. I haven't used bootstrap that much maybe I will check it out. Thanks!!